cloud/amazon/ec2_eni.py

```diff
@@ -29,7 +30,8 @@
     default: null
   instance_id:
     description:
-      - Instance ID that you wish to attach ENI to. To detach an ENI from an instance, use 'None'.
+      - Instance ID that you wish to attach ENI to, if None the new ENI will be created in detached state, existing \
+      ENI will keep current attachment state.
```

Comment:
Maybe change this to...

Instance ID that you wish to attach ENI to.  Since version 2.2, use the 'attached' parameter to attach or detach an ENI.  Prior to 2.2, to detach an ENI from an instance, use 'None'.